src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -76,6 +76,11 @@ private Image(Stream stream)
             throw new NotImplementedException();
         }
 
+            /* Manipulations */
+        public Image Write(string im, string dst)
```

Comment:
Why does it take two strings? I thought we would have extension methods over Image type to manipulate images.